### PR TITLE
Support for Netezza Performance Server as a data source

### PR DESCRIPTION
**Pull request by aniket-s-kulkarni**
## What type of PR is this? 

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## Description
Adding a query runner to support Netezza Performance Server as a data source. 

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [x] Manually
- [ ] N/A

Manual testing included 
- connecting a variety of Netezza backends
- running individual queries
- ensuring that schema discovery is working well

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
New data source

New data source

### CODE REVIEW

**Comment by aniket-s-kulkarni** [comment]
I am confused by the failing check about UI review. Do I need to add a different label for the right review team to take a look at this?

File: requirements.txt
Comment (susodapop) [inline requirements.txt:71]: Query runner dependencies belong in `requirements_all_ds.txt`. `requirements.txt` is only for Redash's own dependencies.

You can see [this article](https://redash.io/help/open-source/dev-guide/write-a-query-runner#Finishing-up) for more info.

```suggestion
```

File: requirements_all_ds.txt
Comment (susodapop) [inline requirements_all_ds.txt:45]: Mirrors what was added to `requirements.txt` originally.

```suggestion
nzpy>=1.15
nzalchemy
```

File: requirements_dev.txt
Comment (susodapop) [inline requirements_dev.txt:16]: `requirements_dev.txt` is only for development dependencies, needed for unit tests and other general development tasks. See earlier comment about where to specify query runner dependencies.

```suggestion
```

**Review by susodapop — CHANGES_REQUESTED** [review]
Thanks so much for this contribution. 

I requested a two small changes to avoid build issues.

I haven't tested this against an active Netezza system. But we'll take your word for it that this succeeds. Happy to merge once the requirements files are fixed.

**Comment by susodapop** [comment]
Our percy check is flaky right now. I've manually verified there are no UI changes that would block merging this PR.

**Comment by aniket-s-kulkarni** [comment]
> Our percy check is flaky right now. I've manually verified there are no UI changes that would block merging this PR.



> Thanks so much for this contribution.
> 
> I requested a two small changes to avoid build issues.
> 
> I haven't tested this against an active Netezza system. But we'll take your word for it that this succeeds. Happy to merge once the requirements files are fixed.

Thanks @susodapop

I have addressed the review comments and 

> Thanks so much for this contribution.
> 
> I requested a two small changes to avoid build issues.
> 
> I haven't tested this against an active Netezza system. But we'll take your word for it that this succeeds. Happy to merge once the requirements files are fixed.


@susodapop thanks, yes I have addressed the review comments and resolved the conflicts. I do have access to Netezza and therefore have been able to test it out.

**Comment by susodapop** [comment]
Thank you! It's nighttime GMT-5. I'll review tomorrow and follow-up with you. Huge thanks for your responsiveness! 🙏

**Review by susodapop — APPROVED** [review]
LGTM

**Pull request merged**
